test/e2e/framework/framework.go

```diff
@@ -272,3 +276,64 @@ func (f *Framework) AfterEach() {
 	}
 	f.OperatorInstalled = false
 }
+
+func printMySQLPodContainerLogs(f *Framework) {
+	pods, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).List(metav1.ListOptions{})
+	if err != nil {
+		Logf("Error retrieving pod list in namespace %s: %+v", f.Namespace.Name, err)
+	}
+
+	var opPod v1.Pod
+	var agPods []v1.Pod
+	for _, pod := range pods.Items {
+		if strings.Contains(pod.Spec.Containers[0].Image, "mysql-operator") {
+			opPod = pod
+			continue
+		}
+		if strings.Contains(pod.Spec.Containers[0].Image, "mysql-agent") || strings.Contains(pod.Spec.Containers[0].Image, "mysql-server") {
+			agPods = append(agPods, pod)
+		}
+	}
+
+	// Operator Logs
+	if opPod.Name != "" {
+		printContainerLogs(f, opPod.GetName(), &v1.PodLogOptions{})
+	}
+
+	for _, agPod := range agPods {
+		// Server Logs
+		printContainerLogs(f, agPod.GetName(), &v1.PodLogOptions{Container: "mysql"})
+		// Agent Logs
+		printContainerLogs(f, agPod.GetName(), &v1.PodLogOptions{Container: "mysql-agent"})
+	}
+}
+
+func printLogs(read io.ReadCloser, fileName string) error {
```

Comment:
Pass in full file path and load `os.Getenv("WERCKER_REPORT_ARTIFACTS_DIR")` on `Framework` initialisation